src/data/strings.json

```
@@ -26,6 +26,26 @@
     "de": "Hergestellt mit",
     "es": "Creado con"
   },
+  "footerAbout": {
+    "en": "About",
+    "pl": "O stronie"
```

These translations are far from direct, but I want them to follow conventions present in these languages. From what I've found, _Unser Projekt_ (de. _Our Project_) will be best for German: https://german.stackexchange.com/questions/26182/what-is-the-german-equivalent-of-about-as-in-about-this-site
```suggestion
    "pl": "O nas",
    "de": "Unser Projekt"
```